src/wxGL_PMFCanvas.cpp

```diff
@@ -395,7 +391,7 @@ void wxGL_PMFCanvas::Render()
 		return;
 
 	IsRendering = true;
-	this->SetCurrent();
+	this->SetCurrent(*m_context);
```

Comment:
This should go in wxGL_PMFCanvas::Init() immediately after m_context = new wxGLContext(this);